js/packages/openinference-instrumentation-openai/src/instrumentation.ts

```diff
@@ -417,8 +417,28 @@ function getChatCompletionInputMessageAttributes(
     [SemanticConventions.MESSAGE_ROLE]: role,
   };
   // Add the content only if it is a string
-  if (typeof message.content === "string")
+  if (typeof message.content === "string") {
```

Comment:
is there no support for output images?

Reply:
currently the chat completion message content is [str only](https://github.com/openai/openai-python/blob/646fff04dbe9458f1c46753031546ccf6415eeef/src/openai/types/chat/chat_completion_message.py#L25-L26)